apps/web-mzima-client/src/app/shared/components/fragments/mobile-menu/mobile-menu.component.scss

```diff
@@ -7,6 +7,7 @@
   bottom: 0;
   z-index: 700;
   position: fixed;
+  overflow-y: scroll;
   color: var(--color-neutral-50);
```

Comment:
Ok, the inverted class works for now 👍🏼

So I see that the share button is still completely hidden, and you have to scroll the entire mobile menu be fore you get to see it.

The share button should be visible without having to scroll get to it @oyenuga17

Reply:
Also, I don't think we need to apply scroll to the entire menu. Since this means at some point you'll have to scroll to get to the close button.

A Preferable solution is where both the close button and the share button are not scrolled out of reach at any point.